CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ All notable changes to the Wazuh app project will be documented in this file.
 - Fixed missing options depending on agent operating system in the agent configuration report [#6983](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6983)
 - Fixed an style that affected the Discover plugin [#6989](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6989)
 - Fixed a problem updating the API host registry in the GET /api/check-stored-api [#6995](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6995)
+- Fixed, when generating a PDF report, the open report button in safari does not work [#7019](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7019)
```

Comment:
```suggestion
- Fixed the `Open report` button does not work on Safari [#7019](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7019)
```

Reply:
I have corrected the changelog by mentioning the two changes I have made [649e7d9](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7019/commits/649e7d9b12dd805cc9fe4f18bd1f976356ee23da)

- Fixed the `Open report` button of the toast and the `Download report` icon of the reporting table in Safari [#7019](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7019)